src/set_theory/cardinal_ordinal.lean

```diff
@@ -4,9 +4,10 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro, Floris van Doorn
```

Comment:
Don't forget to add yourself as an author on this and the other files that you have contributed to